### ISSUE TITLE
Starship emojis are showing as squares

### ISSUE THREAD

**Issue by Saya47** [chrisant996/clink#490]
Hello! Thanks for this great program!
I have a question, I have tried different fonts and utf-8 character page (chcp 65001) but some emoji's still don't show correctly, and they unknown glyph (square character) is shown, what should I do?

Here I'm using Courier new:
![image](https://github.com/chrisant996/clink/assets/35542834/84ebdf9b-302c-432f-babc-761d4439f3f1)

Now I switch to a Nerd font:
![image](https://github.com/chrisant996/clink/assets/35542834/ea7b7122-4727-4590-a3bf-2071771ccba0)

Some emoji's still fail to display (the Python snake emoji):
![image](https://github.com/chrisant996/clink/assets/35542834/7fdec1ca-9d34-4ce0-9e30-7b4e3d3927bb)

Any help is appreciated!

**Comment by chrisant996** [comment]
Thanks for the screenshots, they made it clear what's happening.

You're using the default old conhost terminal in Windows.  It does not support color emoji.  You have to use the new Windows Terminal to get color emoji.

**Comment by chrisant996** [comment]
But also:

You wrote "emoji", and my answer assumes you're indeed referring to emoji.

But if you're instead referring to nerd font icons, then the issue is either the font that's being used or how questions have been answered in the `flexprompt configure` wizard.

**Comment by Saya47** [comment]
Oh I see, thanks very much for answering. Flexprompt configure wizard is brilliant! ✨🎉

**Comment by Saya47** [comment]
Hello @chrisant996 sorry to be a bother again, didn't want to open a new issue because it's a small question, I couldn't find anything on the README or didn't find much on the website about aliases, but from what I've gathered I can use doskey command to make an alias but how do I persist this? I can already see that this key `Computer\HKEY_CURRENT_USER\Software\Microsoft\Command Processor` on Registry is used so I can't use it to make cmd run my alias script so where should I write my scripts? I tried making a bat file on `C:\Users\S\AppData\Local\clink` but it didn't work.
Thanks in advance.
